aiozk/recipes/election.py

```diff
@@ -42,7 +41,8 @@ async def wait_for_leadership(self, timeout=None):
             return
 
         if self.leadership_future is None:
-            self.leadership_future = self.client.loop.create_future()
+            loop = asyncio.get_event_loop()
```

Comment:
Official document says that `asyncio.get_running_loop` is preferred than asyncio.get_event_loop in `coroutines` or `callbacks`.

Reply:
Yeah, but it needs 3.7 version. I've already used `create_task` which is also needs 3.7 and left comment here - https://github.com/micro-fan/aiozk/pull/78#discussion_r579810596. If it's ok to left `create_task` then I'll change all `get_event_loop` to `get_running_loop`.